lib/generateLetters.ml

```diff
@@ -0,0 +1,39 @@
+type letter_deck = char list
+(** letter_deck is a list containing 7 random chars at any time. To be
+    used in a game of scrabble. If the char has been used in the game, a
+    new char will take its place, so there will never be an empty index.
+    Requires each char in the list to be an alphabetical letter at any
+    given time. *)
+
+(** [random_let] is a randomly generated char from a-z. random_let :
+    char*)
+let random_let bound = Random.int bound + 65 |> Char.chr
```

Comment:
```suggestion
let random_let () = Char.chr (Random.int 26 + 65)
```
More traditional way of having a function with no arguments (since every call of this function will assume ```bound = 26```). As a result, change all ```random_let 26``` to ```random_let ()```

Reply:
For some reason this doesn't work - calling random_let results in a value of type unit -> char = <fun> instead of char (as the original function intended) and start_game fills the letter_deck with [<fun>; <fun>; ...] instead of chars like it did when I use bound as a parameter. I agree that that setting 26 to be the upper bound isn't necessary though, open to more suggestions